baselines/pyproject.toml

```diff
@@ -37,13 +37,17 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.8.3"
+python = "^3.7.12"
 # Mandatory dependencies
-flwr = "^0.17.0"
-# flwr = { path = "../", develop = true }  # For development
+#flwr = "^0.17.0"
+flwr = { path = "../", develop = true }  # For development
```

Comment:
Shouldn't this be commented and the line above leave uncommented?

Reply:
I committed the change.